railgun/test/integration/ingress_bulk_test.go

```diff
@@ -0,0 +1,109 @@
+//+build integration_tests
+
+package integration
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kong/kubernetes-ingress-controller/pkg/annotations"
+	"github.com/kong/kubernetes-testing-framework/pkg/utils/kubernetes/generators"
+)
+
+const testBulkIngressNamespace = "ingress-bulk-testing"
+
+func TestIngressBulk(t *testing.T) {
```

Comment:
To be honest, dont quite understand the test cases, do you mind add test step log ?

Reply:
Sure, this log entry should hopefully be more descriptive let me know if you feel this is adequate:
```suggestion
func TestIngressBulk(t *testing.T) {
	t.Log("starting a test to rapidly deploy large numbers of Ingress resources in bulk, to verify functionality at scale")
```

Reply:
Sure, I've added a description to the test in this commit:

49a5db23a14138b655639f8ff6546194ca4163b7

Let me know if you feel this is adequate?